README.md

```
@@ -145,811 +131,78 @@ In order to further reduce bundle size, the modular variant of the SDK performs
-const key = await Ably.Realtime.Crypto.generateRandomKey();
-var channel = client.channels.get('channelName', { cipher: { key: key } });
-
-channel.subscribe(function (message) {
-  message.name; // 'name is not encrypted'
-  message.data; // 'sensitive data is encrypted'
-});
+If you are using this SDK's realtime features, for example, WebSockets in a service worker, note:
 
-channel.publish('name is not encrypted', 'sensitive data is encrypted');
-```
+- In versions of Chrome before 116, active WebSockets would not reset the 30s service worker idle timer, resulting in the client being closed prematurely.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)